source/scss/768.scss

```diff
@@ -1,10 +1,16 @@
 
-.menu {
-    &.main {
-        li {
-            a {
-                font-size: 1.3em;
+header {
+
+  .nav-main {
+   
+    nav {
+   
+      li {
+   
+        a {
+          font-size: 1.3em !important;
```

Comment:
What's this overriding? Can it be removed?

Reply:
as above

Reply:
This is a specificity issue, I've reduced the specificity in `320.scss` and then this is fine, also means we can remove the `!important` in `1200.scss` as well.